src/sparsevector.jl

```diff
@@ -1433,7 +1433,7 @@ for (fun, comp, word) in ((:findmin, :(<), "minimum"), (:findmax, :(>), "maximum
         # we need to find the first zero, which could be stored or implicit
         # we try to avoid findfirst(iszero, x)
         sindex = findfirst(iszero, nzvals) # first stored zero, if any
-        zindex = findfirst(i -> i < nzinds[i], eachindex(nzinds)) # first non-stored zero
+        zindex = findfirst(i -> iszero(x[i]), eachindex(x)) # first non-stored zero
```

Comment:
The comment above says ` # we try to avoid findfirst(iszero, x)`. But isn't this what is done now?

Reply:
I read that comment and I don't know how to avoid that if the objective is to find the first non-stored zero.

I think the original approach is to find the first "hole" between stored value, which would be the first non-stored zero.

I will give it some thought

Reply:
The problem is that indexing into a sparse vector is slow: each time, you need to figure out if the value you're asking for is stored or not. Would be good to check that the fix doesn't spoil the purpose of the specialization, see https://github.com/JuliaLang/julia/issues/42823. If you don't get it "right", it will take a fair amount of time to compute.